docs/topics/api/abuse.rst

```diff
@@ -37,6 +51,20 @@ to if nessecary.
     :>json int|null addon.id: The add-on id on AMO. If the guid submitted didn't match a known add-on on AMO, then null.
     :>json string|null addon.slug: The add-on slug. If the guid submitted didn't match a known add-on on AMO, then null.
     :>json string message: The body/content of the abuse report.
+    :>json string|null addon_install_entry_point: The add-on install entry point.
```

Comment:
These (and below) look duplicated?

Reply:
Those are for the output, above is for the input

Reply:
Eh, indeed… `<` vs `>` - sorry, I didn't see that